IM_calculation/IM/im_calculation.py

```diff
@@ -84,19 +84,19 @@ def check_rotd(comps_to_store: Iterable[Components]) -> bool:
 
 
 def calculate_rotd(
-    spectral_accelerations,
+    accelerations,
     comps_to_store: List[Components],
     func=lambda x: np.max(np.abs(x), axis=1),
 ):
     """
-    Calculates rotd for given spectral accelerations
-    :param spectral_accelerations: An array with shape [periods.size, nt, 2] where nt is the number of timesteps in the original waveform
+    Calculates rotd for given accelerations
+    :param accelerations: An array with shape [periods.size, nt, 2] where nt is the number of timesteps in the original waveform
```

Comment:
Where periods.size is 1 for IMs without period?

Reply:
Yep, the non-pSA IMs give the acceleration array an extra dimension so it fits. 
This line could probably also be changed to [..., [0,1]] for a more robust result.